server/repository/src/migrations/v2_01_00/ledger.rs

```diff
@@ -49,8 +57,8 @@ pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
                 invoice_line.return_reason_id,
                 item_link.item_id AS item_id,
                 CASE
-                    WHEN "type" = 'STOCK_IN' THEN (number_of_packs * pack_size){casting}
-                    WHEN "type" = 'STOCK_OUT' THEN (number_of_packs * pack_size){casting} * -1
+                    WHEN "type" = 'STOCK_IN' THEN (number_of_packs * pack_size)
+                    WHEN "type" = 'STOCK_OUT' THEN (number_of_packs * pack_size) * -1
```

Comment:
@CarlosNZ I had to drop/recreate the same views for this PR. But I tried to keep your migration to indicated what was changed for the ledger...

Could you please check if removing the int casting is correct? or do we want to keep using an int in these views?

Reply:
Should be fine, thanks Clemens 🙏